test/system/pages/careers_page_test.rb

```
@@ -0,0 +1,120 @@
+    visit "/careers/"
+
+    preload_all_images
+
+    # BEHAVIORAL VALIDATION: All 6 benefit cards are visible
+    feature_cards = [
+      "People-Oriented",
+      "Long-Term Focused",
+      "Tight-Knit Team",
+      "Flexible Environment",
+      "Growth Beyond JetThoughts",
+      "World-Class Training"
```

RuboCop (1.81.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)